plugins/woocommerce/src/Internal/DataStores/Orders/OrdersTableDataStore.php

```diff
@@ -948,7 +948,7 @@ public function get_orders_type( $order_ids ) {
 	 */
 	public function get_order_type( $order_id ) {
 		$type = $this->get_orders_type( array( $order_id ) );
-		return $type[ $order_id ];
+		return $type[ $order_id ] ?? '';
```

Comment:
*Not a request for a change,* as such, just an observation: I see a few differences between this and the established CPT implementation.

```
$cpt->get_order_type( $valid_product_id ); # => string(7) "product"
$cot->get_order_type( $valid_product_id ); # => strint(0) ""

$cpt->get_order_type( $invalid_id ); # => bool(false)
$cot->get_order_type( $invalid_id ); # => strint(0) ""
```

I think the approach here is 'more correct' (it correctly satisfies the interface, for one thing), and can't immediately think of a case where this difference will lead to issues, but figured I would note it.

Reply:
Yeah, I think CPT data store mimics the WP_Post behavior. On further thinking after your comment, I removed the return type (`string?`) from the util method declaration, as it wouldn't have worked correctly with CPT store (and would have raised an error) in https://github.com/woocommerce/woocommerce/pull/35349/commits/acebc8dafe1efce9b6618c8c0e16d47f41cf02af. So probably another review is needed.

Reply:
Good idea.